src/utils/PathMap.fs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+/// Functions to map real paths to paths to be written to PDB/IL
+namespace Internal.Utilities
+
+open System
+open System.IO
+
+type PathMap = PathMap of Map<string, string>
+
+[<RequireQualifiedAccess>]
+module internal PathMap =
```

Comment:
Very clear model 👍

Reply:
Thanks Tom - it's a shame that we don't have a better model for file systems. Then this map wouldn't be string to string - alas :)